### ISSUE TITLE
loss function focusing on important errors 

### ISSUE THREAD

**Issue by lyeskhalil** [lyeskhalil/mipGNN#13]
Idea: Setting a low bias threshold for binary classification seems to lead to "better" training. However, this comes off as "hacking" the labels.

Proposal: instead, do regression with an appropriate loss function. The loss function should incur higher cost for underestimating the bias of a variable with true bias non-zero but close to zero.
